tests/adsb_test.py

```diff
@@ -0,0 +1,63 @@
+from pyModeS import adsb
```

Comment:
this is still my question, what if the newest pip release has not included a new function i am adding?